src/domain/pricing/pricing-form/sections/configuration.tsx

```diff
@@ -83,16 +84,17 @@ const Configuration: React.FC<ConfigurationProps> = () => {
               <Controller
                 name="starts_at"
                 control={control}
-                render={({ value, onChange }) => {
+                render={({ field: { value, onChange } }) => {
+                  const ensuredDate = value || new Date()
                   return (
                     <>
                       <DatePicker
-                        date={value}
+                        date={ensuredDate}
                         label="Start date"
                         onSubmitDate={onChange}
                       />
                       <TimePicker
```

Comment:
when selecting dates in the early hours of the morning or late at night the ensuredDate will shift either a day forward or a day back

Reply:
It is still possible to set the dates correctly (you just have to go back and set them again), so lets not take this up right now

Reply:
I think that is just how the DatePicker has always been, since I can't see why the new version of RHF or using the little `ensuredDate` hack would result in this shifting.